requirements.txt

```diff
@@ -32,3 +32,7 @@ requests~=2.32.3
 httpx~=0.27.2
 typing_extensions~=4.12.2
 asgiref~=3.8.1
+
+pytest~=8.3.4
+djangorestframework~=3.15.2
+responses~=0.25.3
```

Comment:
pytest and responses are already provided by `requirements-dev.txt` – we should leave them there since they aren't needed by the production build. `djangorestframework` can also be removed I think, if we use the `pytest-django` provided fixtures for `api_client/admin`.

Reply:
I've removed to other dependencies, but have kept djangorestframework. 
From my observations, pytest-django does not seem to have an api_client fixture. 
So I've maintained my implementation of that. but have removed the admin-client